package/src/renderer/Canvas.tsx

```diff
@@ -101,7 +102,7 @@ export const Canvas = forwardRef<SkiaView, CanvasProps>(
           paint,
           opacity: 1,
           ref,
-          center: Skia.Point(width / 2, height / 2),
+          center: { x: width / 2, y: height / 2 },
```

Comment:
Is this change correct? If yes, can you elaborate?

Reply:
It was to avoid having to create a new Skia Point on every render.